### PR TITLE
Beh 2015 Sequencing example_urban water management + images/tables

### PR DESCRIPTION
**Pull request by Caroline-nc**
Beh sequencing

### CODE REVIEW

File: _examples/beh-2015-sequencing.md [hunk 1]
```diff
@@ -1,12 +1,134 @@
...
+</ul>
+</li>
+</ol>
+</p>
+problem_definition: >
+<p>
+Adelaide, the capital of South Australia, is one of the driest metropolitan centre worldwide, having a Mediterranean climate. It is host of 1.3 million inhabitants with an estimated mean annual water consumption of 163 GL in 2008. The Southern Adelaide water supply system (WSS) is the major water supplier, with delivering 50% of the metropolitan water demand. Three reservoirs support the system supply: Myponga, Mount Bold and Happy Valley; Mount Bold being the major provider. 
+</p>
+<p> 
+To catter for the projected water demand due to population growth and climate change in metropolitan Adelaide, different water supply augmentation sources have been considered including a desalination plant, diverse stormwater harvesting schemes and household rainwater tanks.
```
Comment (leila-n) [inline _examples/beh-2015-sequencing.md:51]: To cater for

File: _examples/beh-2015-sequencing.md [hunk 2]
```diff
@@ -6,64 +6,64 @@ permalink: /examples/beh-2015-sequencing/
 page_authors:
   - Caroline Rosello
   - Joseph Guillaume
-page_reviewers:
-excerpt: The analysis identifies optimal sequences of additional water supply options for a set of scenarios with different climate change, population growth, and discount rates. Sensitivity analysis shows how performance of each plan varies across scenarios, and which variables have greater influence, to help inform selection of an optimal sequence plan.
+page_reviewers: Leila Noble
+excerpt: The analysis identifies optimal sequences for adding to an urban water supply for a set of scenarios with different values for climate change, population and discount rates. Sensitivity analysis shows how the performance of each sequence for multiple criteria varies across the scenarios, and which uncertain variables have greater influence, to help inform decision-makers. 
 approach: >
 <p> 
-The approach consists of the evaluation of the robustness of water supply management plans by considering the sequencing/schedulling of water supply enhancement options (i.e., additional options to support current water supplies) over a planning horizon to cope optimally with future conditions.
```
Comment (josephguillaume) [inline _examples/beh-2015-sequencing.md]: I think robustness is important here. Is there a specific reason it was removed?

File: _examples/beh-2015-sequencing.md [hunk 3]
```diff
@@ -6,64 +6,64 @@ permalink: /examples/beh-2015-sequencing/
 page_authors:
   - Caroline Rosello
   - Joseph Guillaume
-page_reviewers:
-excerpt: The analysis identifies optimal sequences of additional water supply options for a set of scenarios with different climate change, population growth, and discount rates. Sensitivity analysis shows how performance of each plan varies across scenarios, and which variables have greater influence, to help inform selection of an optimal sequence plan.
+page_reviewers: Leila Noble
+excerpt: The analysis identifies optimal sequences for adding to an urban water supply for a set of scenarios with different values for climate change, population and discount rates. Sensitivity analysis shows how the performance of each sequence for multiple criteria varies across the scenarios, and which uncertain variables have greater influence, to help inform decision-makers.
```
Comment (josephguillaume) [inline _examples/beh-2015-sequencing.md]: Need to be specific about "inform decision-makers". Is there a specific reason that "selection of an optimal sequence plan" is incorrect?

File: _examples/beh-2015-sequencing.md [hunk 4]
```diff
@@ -6,64 +6,64 @@ permalink: /examples/beh-2015-sequencing/
...
+<li> identifying plausible future scenarios with different combinations of values of uncertain variables </li>
+<li> generating <a href="../theory/pareto-optimal/">Pareto-optimal </a> sequences for each scenario with respect to pre-defined constraints for the different objective functions. This step may involve the use of multi-objective evolutionary algorithms (MOEAs). </li>
 </ol>
-<li> Step 2 consists of the evaluation of the sensitivity of the Pareto-optimal sequences identified in step 1 to changes in conditions (values) for different uncertain future variables. This step also involves the identification of influential uncertain variables on the sensitivity analysis results. The identifcation of the nature of influential factors would be deterministic on the selection of adaptation measures. Points to consider are: 
+<li> Step 2 consists of a sensitivity analysis. It evaluates the performance the Pareto-optimal sequences identified in step 1 under a range of future conditions and the relative contribution of uncertain variables to the performance. Points to consider are:
```
Comment (josephguillaume) [inline _examples/beh-2015-sequencing.md]: This changes the meaning - evaluation of sensitivity to changes vs evaluating performance under a range of conditions. I think the former is correct for this paper?

Comment (josephguillaume) [reply _examples/beh-2015-sequencing.md]: Actually, from the Figure it does look like it's first the latter, then the former?

File: _examples/beh-2015-sequencing.md [hunk 5]
```diff
@@ -108,24 +109,25 @@ These scenarios are representative of a continuum in future conditions from the
 </p>
 <p> Step 2: Sensitivity analysis. 
 <ul>
-<li> The Sobol's method was used to evaluate the global sensitivity of the objectives to uncertain variables. In addition, the reliability of the supply and the vulnerability of the demand to supply shortfall is estimated. </li> The analysis is repeated 20 times based on 20 sequences of 40 years daily stochastic rainfall sequences and the results of the sensitivity analysis are presented as average values over the 20 sequences. </li>
-<li> Optimal sequence plans' performance to uncertain variables is presented as box-and-whiskers plots displaying the variation of the avarage values for the PV costs, PV GHG emissions, reliability and vulnerability (figure 3) 
+<li> Sobol's method was used for senstivity analysis
+<li> The performance measures were the objectives, reliability (a measure of how often supply exceeds or equals demand) and vulnerability (a measure of demand shortfall). </li> 
+<li> The sensitivity analysis was repeated 20 times using stochastic daily rainfall data for the 40 year planning timeframe. The results were averaged and presented as box-and-whisker plots (figure 3).
```
Comment (josephguillaume) [inline _examples/beh-2015-sequencing.md]: It's not actually the case that the results of the sensitivity analysis was averaged - it's the performance of the plan.

**Comment by josephguillaume** [comment]
Merged, but @leila-n and @Caroline-nc , please let me know if you'd like further changes/fixes by returning to previous wording.

**Pull request merged**